js/src/admin/addSettingsModal.js

```diff
@@ -23,18 +23,18 @@ function populateThemes() {
 
 export default () => {
     app.extensionSettings['fof-nightmode'] = () =>
-        app.modal.show(
-            new SettingsModal({
+        app.modal.show(SettingsModal, {
                 title: app.translator.trans(`${transPrefix}.title`),
                 size: 'small',
                 className: 'fof-nightmode',
-                items: [
+                items: s => [
```

Comment:
I prefer we keep longer parameter names. I believe this one is called `setting` in fof/components? If so we can even save space below by replacing `setting: s,` with `setting,`

Reply:
> I believe this one is called setting in fof/components

It doesn't matter what it is called in fof/components, as this is a function argument.

> If so we can even save space below by replacing setting: s, with setting,

Probably a good idea to extend it here, yeah. I just didn't want to see `setting={setting}` all over the place in the other exts.